int8 forwardDrive = 0
int8 backwardDrive = 1
int8 rightDrive = 2
int8 leftDrive = 3

int8 direction
float64 value

